GVRf/Framework/framework/src/main/jni/objects/components/bone_jni.cpp

```diff
@@ -130,8 +132,10 @@ Java_org_gearvrf_NativeBone_setFinalTransformMatrix(JNIEnv * env, jobject clz, j
 JNIEXPORT jfloatArray JNICALL
 Java_org_gearvrf_NativeBone_getFinalTransformMatrix(JNIEnv * env, jobject clz, jlong ptr) {
     Bone* bone = reinterpret_cast<Bone*>(ptr);
+    glm::mat4 matrix;
 
-    glm::mat4 matrix = bone->getFinalTransformMatrix();
+    if (bone)
+        glm::mat4 matrix = bone->getFinalTransformMatrix();
```

Comment:
You are right. If there is a bone I want to set the matrix, Otherwise it should be identity. I pushed a fix for this.